ui/app/components/child-job-row.hbs

```diff
@@ -0,0 +1,49 @@
+{{!
```

Comment:
Most of this is a copy/paste of the existing <JobRow> component, but because we have sufficient deviation in what we show here, it made more sense for it to be its own component rather than deal with conditionals.